packages/medusa/src/services/event-bus.ts

```diff
@@ -314,6 +316,7 @@ export default class EventBusService {
 
   /**
    * Registers a cron job.
+   * @deprecated All cron job logic has been refactored to the `JobSchedulerService`. This method will be removed in a future release.
```

Comment:
thought: this could just call the `JobScheduler.create` -- then we can remove all the Cron logic in this service.

Reply:
I like it! 🙌